plugin/helpers_bots.go

```diff
@@ -10,6 +10,14 @@ import (
 )
 
 func (p *HelpersImpl) EnsureBot(bot *model.Bot) (retBotId string, retErr error) {
+	const minimumSupportedVersion = "5.10.0"
+	serverVersion := p.API.GetServerVersion()
+
+	err := ensureServerVersion(minimumSupportedVersion, serverVersion)
+	if err != nil {
+		return "", err
```

Comment:
Nit: Maybe use `errors.Wrap` and add the name of the method 0/5

Reply:
ensureServerVersion already sends a descriptive error message. I don't understand how will this be helpful?

Reply:
You could include the name of the function `EnsureBot` to make debugging easier. 0/5